PencilCase/PencilCase/www/index.html

```diff
@@ -16,15 +16,15 @@
 
     <script type="text/javascript" src="cordova.js"></script>
     <!-- The 'async' attribute is needed for RequireJS to work in a Windows app -->
-    <script type="text/javascript" data-main="scripts/startup" src="lib/require.2.1.8.js" async></script>
+    <script type="text/javascript" src="scripts/knockout-3.4.2.js"></script>
     <script type="text/javascript" src="scripts/jquery-1.8.0.min.js"></script>
     <script type="text/javascript" src="scripts/jquery.mobile-1.4.5.min.js"></script>
-    <script type="text/javascript" src="scripts/knockout-3.4.2.js"></script>
```

Comment:
Add knockout js library and moment js library before the require js file, or it will cause some exceptions of not found functionalities.

Reply:
This issue did take some time.
Since without this modification, it worked well with browser simulating, but didn't work when deployed on my Android device.
With this modification, the app works well on both side.